modules/ecr-proxy/alb.tf

```diff
@@ -0,0 +1,60 @@
+#
+# Configure an ALB to be used for this application and associated resources:
+# - alb
+# - ingress security group
+# - listener with default 404 response.
+#
+
+resource "aws_lb" "lb" {
+  name            = "${var.application}-${var.stack}-proxy-lb"
+  subnets         = ["${module.network.private_subnet_ids}"]
+  security_groups = ["${aws_security_group.lb.id}"]
+  internal        = true
+}
+
+output "alb_name" {
+  value = "${aws_lb.lb.dns_name}"
+}
+
+resource "aws_security_group" "lb" {
+  name        = "${var.application}-${var.stack}-proxy-lb-ingress"
+  description = "controls access to the ALB"
+  vpc_id      = "${module.network.vpc_id}"
+  tags        = "${module.tags.application_tags}"
+
+  ingress {
+    protocol    = "tcp"
+    from_port   = "443"
+    to_port     = "443"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
instead of 0.0.0.0/0 (though internal only), would it be worth assigning  module.app.module.circleci.aws_security_group.circleci_services_sg and module.app.module.circleci.module.nomad.aws_security_group.nomad_sg instead just to keep the security a little tighter?